WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagViewController.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+final class SiteTagViewController: UITableViewController {
+    private let blog: Blog
+    private let tag: PostTag
```

Comment:
Just a thought.  PostTag has a `blog` relation.  Would it simplify things a bit to just rely on it instead of having a separate property for blog?

Reply:
Probably. In this case, though, I am expecting either a proper PostTag object, or a stub (in the case when I want to create a new tag). For that second case, I think it will actually be easier if this view controller is aware of blog and tag separately. 

Or maybe not... Would you it be okay if we revisit this discussion until the detail view is done? I would try to remove that extra property and see how the code actually reads

Reply:
Its fine by me to wait til later :)